README.md

```diff
@@ -121,6 +122,16 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [Website](https://bot.land)
 - [Blog](https://blog.bot.land)
 
+### Adron Hall
+#### What Adron Streams
```

Comment:
Could you put your description here so people know what you stream?

Reply:
Whoops, just added!  👍